PyInstaller/cli/opts.py

```diff
@@ -0,0 +1,448 @@
+        '-p',
+        '--paths',
+        multiple=True,
+        type=click.Path(),
+        help='A path to search for imports (like using PYTHONPATH). '
+        f'Multiple paths are allowed, separated by ``{os.pathsep}``, or '
+        'use this option multiple times. Equivalent to '
+        'supplying the ``pathex`` argument in the spec file.'
+    )(cmd)
+    cmd = optgroup('Where to search')(cmd)
+
+    # Group: What to bundle
+
+    cmd = optgroup.option(
+        '--runtime-hook',
+        'runtime_hooks',
+        multiple=True,
+        type=click.File(),
+        help='Path to a custom runtime hook file. A runtime hook is code that '
+        'is bundled with the executable and is executed before any '
+        'other code or module to set up special features of the runtime '
+        'environment. This option can be used multiple times.'
+    )(cmd)
+    cmd = optgroup.option(
+        '--collect-metadata-recursive',
+        multiple=True,
+        metavar='DISTRIBUTIONNAME',
+        help='Recursively collect all metadata from the specified distribution and it\'s dependencies. '
+        'This option can be use multiple times.'
+    )(cmd)
+    cmd = optgroup.option(
+        '--collect-metadata',
+        multiple=True,
+        metavar='DISTRIBUTIONNAME',
+        type=str,
+        help='Collect all metadata from the specified distribution. '
+        'This option can be use multiple times.'
+    )(cmd)
+    cmd = optgroup.option(
+        '--collect-all',
+        multiple=True,
+        metavar='MODULENAME',
+        type=click.UNPROCESSED,
+        callback=lambda _, _p, value: [x for x in value if is_ident(x)],
+        help='Collect all submodules, data files, metadata, and binaries from the specified package or module. '
+        'This option can be use multiple times.'
+    )(cmd)
+    cmd = optgroup.option(
+        '--collect-binaries',
+        multiple=True,
+        metavar='MODULENAME',
+        type=click.UNPROCESSED,
+        callback=lambda _, _p, value: [x for x in value if is_ident(x)],
+        help='Collect all binaries from the specified package or module. '
+        'This option can be use multiple times.'
+    )(cmd)
+    cmd = optgroup.option(
+        '--collect-datas',
+        multiple=True,
+        metavar='MODULENAME',
+        type=click.UNPROCESSED,
+        callback=lambda _, _p, value: [x for x in value if is_ident(x)],
+        help='Collect all data files from the specified package or module. '
+        'This option can be use multiple times.'
+    )(cmd)
+    cmd = optgroup.option(
+        '--collect-submodules',
+        multiple=True,
+        metavar='MODULENAME',
+        type=click.UNPROCESSED,
+        callback=lambda _, _p, value: [x for x in value if is_ident(x)],
+        help='Collect all submodules from the specified package or module. '
+        'This option can be used multiple times.'
+    )(cmd)
+    cmd = optgroup.option(
+        '--excluded-import',
```

Comment:
I think I liked the original `--exclude-module` better.

Reply:
I think I may have made a mistake here... Will check when on laptop. Don't remember intending to change this option name.